src/askscript/__tests__/00-documentation-examples/documentation01-complete_example.test.ask

```diff
@@ -1,9 +1,7 @@
 ask {
-  test('hello', fun () {
```

Comment:
Why not keep the `test` resource - It's a standard of Jest to group expectations into tests

Reply:
You could optionally use `createEmptyAssertionResult`, `passed`, `failed` function to mark individual tests. Let's keep it easy as this is a prototype integration between AskScript <-> Jest, however, let's support all the basic blocks such us `test`, `expect` and `toBe` so that we can start writing AskScript tests already and then figure out better Jest integration

Reply:
> Why not keep the `test` resource - It's a standard of Jest to group expectations into tests

@mhagmajer , I tried to add the Jest's test resource, but failed. See https://github.com/xFAANG/askql/pull/271#issue-440937327 for details.

Reply:
> You could optionally use `createEmptyAssertionResult`, `passed`, `failed` function to mark individual tests. Let's keep it easy as this is a prototype integration between AskScript <-> Jest, however, let's support all the basic blocks such us `test`, `expect` and `toBe`/`toEqual` etc so that we can start writing AskScript tests already and then figure out better Jest integration

Current PR is enough for writing AskScript tests, see screenshot at https://github.com/xFAANG/askql/pull/271#issue-440937327

Reply:
@czerwinskilukasz1 can you add very simple implementation of `test`? I'd like AskScript tests to group tests same way like TypeScript does. It can have very simple implementation like:

```
let test = fun(name, f) { f() }
```

Reply:
@mhagmajer , I guess we don't have a good way to define resources in AskScript, do we?

Reply:
I think I don't understand the way askvm calls functions well enough. 
I implemented `test` as:
```
test: resource({
      type: any,
      async compute(options, code, args) {
        assert(
          typeof args !== 'undefined' && args.length == 2,
          'test() expects exactly two arguments'
        );
        const fun = args[1] as TypedValue<AskCode>;
        return call.compute(options, fun.value);
      },
    }),
```

and I'm getting 'Cannot call null'.
Any tips?